assets/javascripts/modules/analytics/ga.js

```diff
@@ -16,6 +18,10 @@ define(['utils/cookie',
         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
         /*eslint-enable */
 
+        // Do JellyFish first as they don't want our custom dimensions
+        ga('create', 'UA-44575989-1', 'auto', 'jellyfishGA');
```

Comment:
Since we have 2 trackers - ours and jellyfish can we explicitly call our tracker on the custom dimension sets.
This way we can have the pageview sends together at the end.